tests/test_transferlearning.py

```diff
@@ -0,0 +1,247 @@
+import numpy as np
+from numpy.testing import assert_array_equal
+from pyriemann.transferlearning import MDWM
+import pytest
+
+
+# rclf = [MDM, FgMDM, KNearestNeighbor, TSclassifier]
+mat_transfer = [MDWM]
+
+
+@pytest.mark.parametrize("transfer", mat_transfer)
+class TransferTestCase:
+    def test_transfer_with_target_sample(self, transfer,
+                                         get_covmats, get_labels):
+        n_classes, n_channels = 2, 3
+        n_matrices_class_source, n_matrices_class_target = 3, 2
+        n_matrices_source = n_classes * n_matrices_class_source
+        n_matrices_target = n_classes * n_matrices_class_target
+        sample_weight = None
+
+        covmats_source = get_covmats(n_matrices_source, n_channels)
+        labels_source = get_labels(n_matrices_source, n_classes)
+
+        covmats_target = get_covmats(n_matrices_target, n_channels)
+        labels_target = get_labels(n_matrices_target, n_classes)
+
+        if transfer is MDWM:
+            self.trans_populate_classes(transfer, covmats_source,
+                                        labels_source, covmats_target,
+                                        labels_target, sample_weight)
+            self.trans_predict(transfer, covmats_source, labels_source,
+                               covmats_target, labels_target, sample_weight)
+
```

Comment:
```suggestion
            self.trans_jobs(transfer, covmats_source, labels_source,
                               covmats_target, labels_target, sample_weight)
```

You could take inspiration from `test_classification.py` to add `trans_predict_proba`, `trans_fitpredict` like their `clf_*` counterparts.